### PR TITLE
[6.4] UI Port test_packages to airgun

### PR DESCRIPTION
**Pull request by ldjebran**
depend on https://github.com/SatelliteQE/airgun/pull/169
```bash
(robottelo) dlezz@dlezz:~/projects/robottelo-fork$ pytest -v -n 4 tests/foreman/ui_airgun/test_package.py 
============================================ test session starts =============================================
platform linux -- Python 3.6.6, pytest-3.6.1, py-1.5.4, pluggy-0.6.0 -- /home/dlezz/.pyenv/versions/3.6.6/envs/robottelo/bin/python
cachedir: .pytest_cache
shared_function enabled - ON - scope: ffff - storage: file
rootdir: /home/dlezz/projects/robottelo-fork, inifile:
plugins: wait-for-1.0.9, xdist-1.22.5, services-1.2.1, mock-1.6.3, forked-0.2, cov-2.5.1
[gw0] linux Python 3.6.6 cwd: /home/dlezz/projects/robottelo-fork
[gw1] linux Python 3.6.6 cwd: /home/dlezz/projects/robottelo-fork
[gw2] linux Python 3.6.6 cwd: /home/dlezz/projects/robottelo-fork
[gw3] linux Python 3.6.6 cwd: /home/dlezz/projects/robottelo-fork
[gw0] Python 3.6.6 (default, Jul 31 2018, 17:47:38)  -- [GCC 5.4.0 20160609]
[gw1] Python 3.6.6 (default, Jul 31 2018, 17:47:38)  -- [GCC 5.4.0 20160609]
[gw2] Python 3.6.6 (default, Jul 31 2018, 17:47:38)  -- [GCC 5.4.0 20160609]
[gw3] Python 3.6.6 (default, Jul 31 2018, 17:47:38)  -- [GCC 5.4.0 20160609]
gw0 [5] / gw1 [5] / gw2 [5] / gw3 [5]
scheduling tests via LoadScheduling

tests/foreman/ui_airgun/test_package.py::test_positive_search_in_multiple_repos 
tests/foreman/ui_airgun/test_package.py::test_positive_search_in_repo 
tests/foreman/ui_airgun/test_package.py::test_positive_check_package_details 
tests/foreman/ui_airgun/test_package.py::test_positive_check_custom_package_details 
[gw0] [ 20%] PASSED tests/foreman/ui_airgun/test_package.py::test_positive_search_in_repo 
tests/foreman/ui_airgun/test_package.py::test_positive_rh_repo_search_and_check_file_list 
[gw2] [ 40%] PASSED tests/foreman/ui_airgun/test_package.py::test_positive_check_package_details 
[gw3] [ 60%] PASSED tests/foreman/ui_airgun/test_package.py::test_positive_check_custom_package_details 
[gw1] [ 80%] PASSED tests/foreman/ui_airgun/test_package.py::test_positive_search_in_multiple_repos 
[gw0] [100%] PASSED tests/foreman/ui_airgun/test_package.py::test_positive_rh_repo_search_and_check_file_list 

======================================== 5 passed in 1236.19 seconds =========================================

```

### CODE REVIEW

**Comment by ldjebran** [comment]
test run output to latest changes
```bash
(robottelo) dlezz@dlezz:~/projects/robottelo-fork$ pytest -v tests/foreman/ui_airgun/test_package.py
============================================ test session starts =============================================
platform linux -- Python 3.6.6, pytest-3.6.1, py-1.5.4, pluggy-0.6.0 -- /home/dlezz/.pyenv/versions/3.6.6/envs/robottelo/bin/python
cachedir: .pytest_cache
shared_function enabled - ON - scope: ffff - storage: file
rootdir: /home/dlezz/projects/robottelo-fork, inifile:
plugins: wait-for-1.0.9, xdist-1.22.5, services-1.2.1, mock-1.6.3, forked-0.2, cov-2.5.1
collecting 5 items                                                                                           2018-08-07 16:21:23 - conftest - DEBUG - BZ deselect is disabled in settings

collected 5 items                                                                                            

tests/foreman/ui_airgun/test_package.py::test_positive_search_in_repo PASSED                           [ 20%]
tests/foreman/ui_airgun/test_package.py::test_positive_search_in_multiple_repos PASSED                 [ 40%]
tests/foreman/ui_airgun/test_package.py::test_positive_check_package_details PASSED                    [ 60%]
tests/foreman/ui_airgun/test_package.py::test_positive_check_custom_package_details PASSED             [ 80%]
tests/foreman/ui_airgun/test_package.py::test_positive_rh_repo_search_and_check_file_list PASSED       [100%]

======================================== 5 passed in 3091.03 seconds =========================================
```

File: tests/foreman/ui_airgun/test_package.py [hunk 1]
```diff
@@ -0,0 +1,250 @@
...
+
+    :id: e182a89f-74e4-4b29-8152-1ea3bd014fd3
+
+    :expectedresults: Content search functionality works as intended and
+        expected packages are present inside of repository
+
+    :CaseLevel: Integration
+    """
+    with session:
+        session.organization.select(org_name=module_org.name)
```
Comment (abalakh) [inline tests/foreman/ui_airgun/test_package.py]: This line should be redundant here and in all the rest tests - module_user which is used in tests has default organization set to `module_org`. Basically, 2 main ideas behind `module_org` is to have 1 default org per module and to have that org automatically selected :)

File: tests/foreman/ui_airgun/test_package.py [hunk 2]
```diff
@@ -0,0 +1,250 @@
...
+        assert session.package.search(
+            'name = bear', repository=module_yum_repo.name
+        )[0]['RPM'].startswith('bear')
```
Comment (abalakh) [inline tests/foreman/ui_airgun/test_package.py]: I was using `FAKE_0_CUSTOM_PACKAGE_NAME` and `FAKE_0_CUSTOM_PACKAGE` in my tests (`bear`/`bear-4.1-1.noarch`). Same with FAKE_1_CUSTOM_PACKAGE_NAME/FAKE_1_CUSTOM_PACKAGE. Maybe it's worth following the same approach if applicable? Not a blocker

File: tests/foreman/ui_airgun/test_package.py [hunk 3]
```diff
@@ -0,0 +1,250 @@
...
+            'build_Host': 'smqe-ws15',
+            'build_time': '1331831364',
+        }
+        all_package_details = session.package.read(
+            'gorilla', repository=module_yum_repo.name)['details']
+        package_details = {
+            key: value for key, value in all_package_details.items()
+            if key in expected_package_details
+        }
+        assert expected_package_details, package_details
```
Comment (abalakh) [inline tests/foreman/ui_airgun/test_package.py]: isn't that a typo? did you mean `assert expected_package_details == package_details`? i'm afraid your assertion will always pass

File: tests/foreman/ui_airgun/test_package.py (hunk 1 above)
Comment (ldjebran) [reply tests/foreman/ui_airgun/test_package.py]: Prefer to be sure that the test is in that org

**Comment by ldjebran** [comment]
@abalakh fixed that assertion expression, run all the tests again with the master airgun
```bash
(robottelo) dlezz@dlezz:~/projects/robottelo-fork$ pytest -v tests/foreman/ui_airgun/test_package.py -v
============================================ test session starts =============================================
platform linux -- Python 3.6.6, pytest-3.6.1, py-1.5.4, pluggy-0.6.0 -- /home/dlezz/.pyenv/versions/3.6.6/envs/robottelo/bin/python
cachedir: .pytest_cache
shared_function enabled - ON - scope: ffff - storage: file
rootdir: /home/dlezz/projects/robottelo-fork, inifile:
plugins: wait-for-1.0.9, xdist-1.22.5, services-1.2.1, mock-1.6.3, forked-0.2, cov-2.5.1
collecting 5 items                                                                                           2018-08-15 10:38:34 - conftest - DEBUG - BZ deselect is disabled in settings

collected 5 items                                                                                            

tests/foreman/ui_airgun/test_package.py::test_positive_search_in_repo PASSED                           [ 20%]
tests/foreman/ui_airgun/test_package.py::test_positive_search_in_multiple_repos PASSED                 [ 40%]
tests/foreman/ui_airgun/test_package.py::test_positive_check_package_details PASSED                    [ 60%]
tests/foreman/ui_airgun/test_package.py::test_positive_check_custom_package_details PASSED             [ 80%]
tests/foreman/ui_airgun/test_package.py::test_positive_rh_repo_search_and_check_file_list PASSED       [100%]

========================================= 5 passed in 326.18 seconds =========================================
```

File: tests/foreman/ui_airgun/test_package.py (hunk 1 above)
Comment (abalakh) [reply tests/foreman/ui_airgun/test_package.py]: no point in using module_org then. anyway, not a blocker

**Pull request merged**